design/frontend/template/autocomplete/product.phtml

```diff
@@ -1,32 +1,16 @@
-<?php
-
-/** @var Algolia_Algoliasearch_Helper_Config $config */
-$config = Mage::helper('algoliasearch/config');
-
-/** @var Mage_Customer_Model_Session $session */
-$session = Mage::getSingleton('customer/session');
-$group_id = $session->getCustomerGroupId();
-
-$currency_code = Mage::app()->getStore()->getCurrentCurrencyCode();
-$price_key = $config->isCustomerGroupsEnabled(Mage::app()->getStore()->getStoreId()) ? '.'.$currency_code.'.group_'.$group_id : '.'.$currency_code.'.default';
-
-$image_base_url = Mage::getBaseUrl();
-
-?>
-
 <!-- Product hit template -->
 <script type="text/template" id="autocomplete_products_template">
     <a class="algoliasearch-autocomplete-hit" href="{{url}}">
         {{#thumbnail_url}}
-        <div class="thumb"><img src="<?php echo $image_base_url; ?>{{thumbnail_url}}" /></div>
+        <div class="thumb"><img src="{{{ baseUrl }}}/{{ thumbnail_url }}" /></div>
         {{/thumbnail_url}}
 
         <div class="info">
             {{{_highlightResult.name.value}}}
 
             <div class="algoliasearch-autocomplete-category">
                 {{#categories_without_path}}
-                <?php echo $this->__('in'); ?> {{{categories_without_path}}}
+                {{ inTranslation }} {{{categories_without_path}}}
```

Comment:
Well I wanted to remove all PHP dependency as first intention was put everything to JS files. There is bunch of similar occurences of that. So now it depends how we decide to proceed.